com.makemkv.MakeMKV.yaml

```diff
@@ -0,0 +1,112 @@
+app-id: com.makemkv.MakeMKV
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
```

Comment:
Have you checked that MakeMKV works with the new version? I have updated OpenJDK in #29, which lead to problems and therefore was reverted in #36.

Reply:
I have been running this version for a while now. By which i mean the test build. Seemed fine so far for me. And menus were correctly extracted.

Reply:
> I have been running this version for a while now. By which i mean the test build. Seemed fine so far for me. And menus were correctly extracted.

Thanks for reporting back that the build has been working.